python/lsst/obs/base/mapper_tests.py

```diff
@@ -0,0 +1,207 @@
+                     map_python_type=None,
+                     map_cpp_type=None,
+                     map_storage_name=None,
+                     raw_filename=None,
+                     default_level=None,
+                     raw_levels=None,
+                     ):
+        """
+        Set up the necessary variables for mapper tests.
+
+        output: str
+            full path to output repository (can be the same as data_dir input repository)
+        path_to_raw: str
+            full path to the raw file referenced by dataIds['raw']
+        keys: set
+            dictionary keys that this mapper should contain
+        query_format: list
+            format list for the results portion of queryMetadata
+        queryMetadata: tuple of (dict, tuple)
+            dataIds and the results of calling them in queryMetadata
+        metadata_output_path: str
+            path to metadata output associated with dataIds['raw']
+        map_python_type: str
+            full python type specification returned by the mapper for dataIds['raw']
+        map_cpp_type: str
+            C++ type specification returned by the mapper for dataIds['raw']
+        map_storage_name: str
+            butler name for the storage type dataIds['raw']
+        raw_filename: str
+            Name of the raw files returned by the mapper for dataIds['raw']
+        default_level: str
+            value returned from mapper.getDefaultLevel
+        raw_levels: tuple of (str, set of str)
+            (level, expect) level and expected mapper return for mapper.getKeys('raw', level)
+        """
+        fields = ['output',
+                  'path_to_raw',
+                  'keys',
+                  'query_format',
+                  'queryMetadata',
+                  'metadata_output_path',
+                  'map_python_type',
+                  'map_cpp_type',
+                  'map_storage_name',
+                  'raw_filename',
+                  'default_level',
+                  'raw_levels',
+                  ]
+        MapperData = collections.namedtuple("MapperData", fields)
+        self.mapperData = MapperData(output=output,
+                                     path_to_raw=path_to_raw,
+                                     keys=keys,
+                                     query_format=query_format,
+                                     queryMetadata=queryMetadata,
+                                     metadata_output_path=metadata_output_path,
+                                     map_python_type=map_python_type,
+                                     map_cpp_type=map_cpp_type,
+                                     map_storage_name=map_storage_name,
+                                     raw_filename=raw_filename,
+                                     default_level=default_level,
+                                     raw_levels=raw_levels,
+                                     )
+
+    def tearDown(self):
+        del self.mapper
```

Comment:
`self.mapper` isn't being set in the this class, so shouldn't be destroyed by this class.